tests/basic/misc_test.go

```diff
@@ -232,6 +321,11 @@ var _ = Describe("{VolumeDriverCrash}", func() {
 			}
 		})
 
+		if pureCleanupFunction != nil {
```

Comment:
nit: why is it necessary to introduce this pureCleanupFunction variable?
For readability purpose I guess we could introduce a boolean like `isDirectAccessVolume`.. not seeing clear benefit of `pureCleanupFunction` here since looks like Inst().V.String() == portworx.PureDriverName is equivalent to pureCleanupFunction != nil and is arguably easier to follow.